lib/auth/sessions.go

```diff
@@ -233,3 +233,26 @@ func (s *Server) createSessionCert(user types.User, sessionTTL time.Duration, pu
 
 	return certs.SSH, certs.TLS, nil
 }
+
+// deleteWebSessions deletes all web sessions for the specified user.
+func (s *Server) deleteWebSessions(ctx context.Context, username string) error {
+	websessions, err := s.Identity.WebSessions().List(ctx)
```

Comment:
Does this list _all_ web sessions? Won't that be a huge number in large clusters?

If yes, is it possible to search only for the user sessions?

Reply:
> Does this list all web sessions? Won't that be a huge number in large clusters?

I did get an opinion from @russjones where since this call does not happen often, that it might be okay

> If yes, is it possible to search only for the user sessions?

Not with the type of key we have: https://github.com/gravitational/teleport/blob/master/lib/services/local/session.go#L312
It looks like we used to store by user, which would've made it possible to query by `users/<username>/*`, not sure why we decided to change it

Reply:
Ack. Thanks for looking it up.

Reply:
Could you please leave a comment/TODO there as a warning?